sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -7,8 +7,10 @@
 #### Breaking Changes
 
 #### Bugs Fixed
+* Fixed bug with non english locales causing an error with the RFC 1123 Date Format. See [PR 30125](https://github.com/Azure/azure-sdk-for-python/pull/30125).
 
 #### Other Changes
+* Refactoring of our client config settings - `connection_timeout` and `request_timeout` are now considered for different configurations. See [PR 30171](https://github.com/Azure/azure-sdk-for-python/pull/30171).
```

Comment:
I'm not sure I understand this. Is the behaviour here being changed?

Reply:
IIRC, `request_timeout` was the legacy parameter for what was replaced by `connection_timeout` for track2/AzureCore pipeline. So I don't think these were intended to be different/separate configurations.

Reply:
We can't remove support for the parameter, but we can remove the documentation for it so that new customers start using the correct parameter, and aren't confused by seeing multiple documented options.

Reply:
We should probably make sure we have some inline comments in the code so that future developers don't remove support for `request_timeout` if they don't see it documented anywhere.

Reply:
I went ahead and made the documentation changes as well as adding a comment for clarity.